### ISSUE TITLE
--skip_unknown_resource_types flag to avoid "unknown resource type" panic for `--resources=*`

### ISSUE THREAD

**Issue by brunobowden** [GoogleCloudPlatform/terraformer#652]
When attempting to export all resources from a GCP project, the export failed with the eror "unknown resource type": `google_compute_managed_ssl_certificate`. Though it would be good to add support for that type (#653)... this request is to add a flag to allow skipping that unknown resources and continuing with the export, e.g. `--skip_unknown_resource_types` or similar.

It's probably still appropriate for it to fail by default without the flag. The intent of using `--resources=*` was to see all the resources in the project and not need to specify a large list of resources that was difficult to figure out. When it panic fails, terraformer should mention the flag, which will likely be useful for the developer to know.

As a minor issue, the prompts to enable APIs are helpful. If possible, it would be nice to batch them together, so you don't need to rerun the command to find the next missing API... but this is more of a minor issue.

## Logs

```
$ terraformer --version
version v0.8.9
$ terraformer import google --regions=us-central1 --compact --projects=who-myhealth-staging --resources=*
...
2020/11/04 15:03:03 google importing... sslCertificates
2020/11/04 15:03:04 Refreshing state... google_compute_managed_ssl_certificate.tfer--who-002D-myhealth-002D-cert-002D-brunos
panic: unknown resource type google_compute_managed_ssl_certificate

goroutine 1521 [running]:
github.com/hashicorp/terraform/plugin.(*GRPCProvider).getResourceSchema(0xc000770230, 0x80f9c9f, 0x26, 0x0, 0x0)
	github.com/hashicorp/terraform@v0.12.29/plugin/grpc_provider.go:92 +0x12a
github.com/hashicorp/terraform/plugin.(*GRPCProvider).ReadResource(0xc000770230, 0x80f9c9f, 0x26, 0x88792c0, 0xc00107ad38, 0x6e26ac0, 0xc0020818f0, 0xd3d9b50, 0x0, 0x0, ...)
	github.com/hashicorp/terraform@v0.12.29/plugin/grpc_provider.go:321 +0xa5
github.com/GoogleCloudPlatform/terraformer/terraformutils/providerwrapper.(*ProviderWrapper).Refresh(0xc001447f80, 0xc00163cf50, 0xc00163cf00, 0x2, 0xc000fe4690, 0x69)
	github.com/GoogleCloudPlatform/terraformer/terraformutils/providerwrapper/provider.go:153 +0x28b
github.com/GoogleCloudPlatform/terraformer/terraformutils.(*Resource).Refresh(0xc001842380, 0xc001447f80)
	github.com/GoogleCloudPlatform/terraformer/terraformutils/resource.go:122 +0x4f
github.com/GoogleCloudPlatform/terraformer/terraformutils.RefreshResourceWorker(0xc000f968a0, 0xc00129b280, 0xc001447f80)
	github.com/GoogleCloudPlatform/terraformer/terraformutils/utils.go:107 +0xcd
created by github.com/GoogleCloudPlatform/terraformer/terraformutils.RefreshResources
	github.com/GoogleCloudPlatform/terraformer/terraformutils/utils.go:77 +0x108
```

**Comment by meshuga** [comment]
Thanks for the issue. In recent issues #627 and #607 we enhanced error recovery in terraformer and now we apply _best effort_ approach to resource discovery. I hope that'll cover all of the cases of the tool users

**Comment by brunobowden** [comment]
@meshuga - thanks for the comment. I'll mark this closed due to #628.

I'll confirm either way when a new version is available.
